.rubocop.yml

```diff
@@ -875,6 +877,76 @@ Metrics/PerceivedComplexity:
 Migration/DepartmentName:
   Enabled: true
 
+# Failing Minitest cops - disabled pending dev review
```

Comment:
I'm a bit surprised to see all of these disabled! Does that mean all the Minitest cops are failing with our codebase right now?

Reply:
(I have a similar question for Rake! I'm not familiar with these gems 😄)